cumulusci/cumulusci.yml

```diff
@@ -409,6 +409,9 @@ tasks:
         options:
             mapping: 'datasets/mapping.yml'
             sql_path: 'datasets/sample.sql'
+    upload_custom_settings:
+        description: Upload Custom Settings specified in a YAML file to the target org
+        class_path: cumulusci.tasks.salesforce.UploadCustomSettings
```

Comment:
Please set a `group` for each new task to control where it appears in `cci task list`. Maybe a new "Data" group that includes this as well as the bulk data tasks?